src/site/content/en/blog/hid/index.md

```diff
@@ -0,0 +1,478 @@
+behavior of specific devices.
+
+## Current status {: #status }
+
+<div class="w-table-wrapper">
+
+| Step                                         | Status                       |
+| -------------------------------------------- | ---------------------------- |
+| 1. Create explainer                          | [Complete][explainer]        |
+| 2. Create initial draft of specification     | [In Progress][spec]          |
+| **3. Gather feedback & iterate on design**   | [**In Progress**](#feedback) |
+| **4. Origin trial**                          | [**In Progress**](#ot)       |
+| 5. Launch                                    | Not started                  |
+
+</div>
+
+## Terminology {: #terminology }
+
+HID consists of two fundamental concepts: reports and report descriptors.
+Reports are the actual data that is exchanged between a device and a software
+client. The report descriptor describes the format and meaning of data that the
+device supports.
+
+A HID (Human Interface Device) is a type of device that takes input from or
+provides output to humans. It also refers to the HID protocol, a standard for
+bi-directional communication between a host and a device that is designed to
+simplify the installation procedure. The HID protocol was originally developed
+for USB devices, but has since been implemented over many other protocols,
+including Bluetooth.
+
+Applications and HID devices exchange binary data through reports. There are
+three report types:
+
+<div class="w-table-wrapper">
+  <table>
+    <thead>
+      <tr>
+        <th>Report type</th>
+        <th>Description</th>
+      </tr>
+    </thead>
+    <tbody>
+      <tr>
+        <td>Input&nbsp;report</td>
+        <td>Data that is sent from the device to the application (e.g. a button is pressed.)</td>
+      </tr>
+      <tr>
+        <td>Output&nbsp;report</td>
+        <td>Data that is sent from the application to the device (e.g. request to turn on the keyboard backlight.)</td>
+      </tr>
+      <tr>
+        <td>Feature&nbsp;report</td>
+        <td>Data that may be sent in either direction. Format is device-specific.</td>
+      </tr>
+    </tbody>
+  </table>
+</div>
+
+A report descriptor describes the binary format of reports supported by the
+device. Its structure is hierarchical and can group reports together as distinct
+collections within the top-level collection. The [format] of the descriptor is
+defined by the HID specification.
+
+A HID usage is a numeric value referring to a standardized input or output.
+Usage values allow a device to describe the intended use of the device and the
+purpose of each field in its reports. For example, one is defined for the left
+button of a mouse. Usages are also organized into usage pages, which provide an
+indication of the high-level category of the device or report.
+
+## Using the WebHID API {: #use }
+
+### Enabling support during the origin trial phase
```

Comment:
I suggest the article instruct flag usage first (simpler) and then why and how to use an Origin Trial.

Reply:
@petele @tomayac What do you think of this suggestion? Shall we change our fugu doc template?

Reply:
👍 I'm not opposed. Most developers, even the ones who end up going the OT route, will likely first test locally with a flag flipped.

Reply:
I've noticed afterwards you actually moved it in your recent articles. Let's update our template then.

Reply:
Fixed in https://github.com/GoogleChrome/web.dev/pull/3860/commits/61184ac69b90e61c13f54d06ac074010b077b1a5